src/darwin/Framework/CHIP/MTRConversion.h

```diff
@@ -38,4 +40,10 @@ inline NSDate * ChipEpochSecondsAsDate(uint32_t chipEpochSeconds)
     return [NSDate dateWithTimeIntervalSince1970:(chip::kChipEpochSecondsSinceUnixEpoch + (NSTimeInterval) chipEpochSeconds)];
 }
 
+/**
+ * Utilities for converting between NSSet<NSNumber *> and chip::CATValues.
+ */
+CHIP_ERROR ToCATValues(NSSet<NSNumber *> * catSet, chip::CATValues & values);
```

Comment:
The naming of our other utilities is `AsBar` or `FooAsBar`, might be worthwhile to follow the same convention here, e.g. `AsCATValues` and `CATValuesAsSet` ?

Reply:
Hm but then those don't return CHIP_ERROR either...

Reply:
I considered that, but it looks a bit weird with the CHIP_ERROR return type....